README.md

```diff
@@ -10,24 +10,23 @@ Camera, Scene, Renderers, loaders you may use to load 3D objects.
 
 # Installation
 
-First make sure that you have `kivy` installed on your machine. Mostly
-a simple:
+Create a virtual environment.
 
-    $ pip install kivy
+    $ python3 -m venv .
 
-should be sufficient, but stable version will not include the latest features.
-For further info visit
-[Kivy documentation](https://kivy.org/docs/installation/installation.html).
-After successful Kivy installation install Kivy3 with this:
+Install dependencies.
 
-    $ pip install https://github.com/KeyWeeUsr/kivy3/zipball/master
+    $ ./bin/pip install Cython==0.29.9
+    $ ./bin/pip install kivy==1.11.1
+    $ ./bin/pip install numpy
+    $ ./bin/pip install scipy
+    $ ./bin/pip install numpy-stl
+    $ ./bin/pip install https://github.com/ros-infrastructure/catkin_pkg/archive/0.4.17.tar.gz
+    $ ./bin/pip install https://github.com/ros/urdf_parser_py/archive/0.4.3.tar.gz
```

Comment:
This usually goes to the setup.py https://github.com/kivy/kivy3/blob/12b99ff/setup.py#L26 so it gets pulled automatically as dependency